src/lib/tooltip.tsx

```diff
@@ -0,0 +1,48 @@
+type PositionProps = 'bottom' | 'top' | 'left' | 'right';
+
+interface GetPositionProps {
+  id: string;
+  position: PositionProps;
+}
+
+const positions: { [key: string]: PositionProps } = {
+  top: 'bottom',
+  bottom: 'top',
+  left: 'right',
+  right: 'left',
+};
+
+export const getPosition = ({ position, id }: GetPositionProps) => {
```

Comment:
Давай попробуем именовать функции и методы в таком формате:

https://gist.github.com/sergeysova/9f6dc24e21efe5f5bb4f9878216c2ce9

```suggestion
export const positionRelativeGet = ({ position, id }: GetPositionProps) => {
```